components/table/table-wrapper.js

```diff
@@ -262,6 +268,12 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 				--d2l-table-cell-padding-alt: calc(0.6rem - 1px) 0.6rem 0.6rem 0.6rem;
 				--d2l-table-border-color: var(--d2l-color-gypsum);
 				--d2l-table-header-background-color: #ffffff;
+				--d2l-sortable-button-border-radius: 0.2rem;
+				--d2l-sortable-button-padding: 0.6rem;
+				--d2l-sortable-button-height: calc(var(--d2l-table-cell-height) * 0.9);
```

Comment:
The height of the hovered light items looks quite tall as compared to the spec. Did the spec get changed?

Reply:
I modified it to use the spec height of 40px